### ISSUE TITLE
WalletRepository

### ISSUE THREAD

**Issue by refactorthis** [carable/carable.assemblypayments#19]
Hi 

Thanks for sharing. 
Looking for wallet withdrawal functionality, can't see it, looks like this hasn't been implemented?

Would you accept a pr for this?

**Comment by wallymathieu** [comment]
Talk to @plesko about getting pull request accepted. You can also talk to Assembly payment about creating a Assembly Payment .Net core i library since it is a port of an existing library to .Net core style.

**Comment by refactorthis** [comment]
Yeah im aware, not interested in creating a new library,  just extending yours as it works well for our purposes short of that missing repository.

**Comment by wallymathieu** [comment]
I meant that [AssemblyPayments](https://github.com/AssemblyPayments) could fork the code (or merge the changes back into [promisepay-dotnet](https://github.com/AssemblyPayments/promisepay-dotnet) ) and publish it under AssemblyPayments prefix (since that's their new name).

**Comment by refactorthis** [comment]
Pull request for change #20

Will perform some tests this week before this is to be merged.
